client/web/src/search/home/SearchPage.tsx

```diff
@@ -78,8 +80,28 @@ export const SearchPage: React.FunctionComponent<React.PropsWithChildren<SearchP
                     </div>
                 </div>
             )}
+
             <div className={styles.searchContainer}>
-                <SearchPageInput {...props} queryState={queryState} setQueryState={setQueryState} source="home" />
+                {shouldShowAddCodeHostWidget ? (
+                    <>
+                        <Tooltip
+                            content="Sourcegraph is not fully functional until a code-host is set up"
```

Comment:
I've never seen this spelled with a dash.

```suggestion
                            content="Sourcegraph is not fully functional until a code host is set up"
```

Reply:
@vdavid, I'm not sure about this, as [I do NOT see the usage of hyphens](https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+code+host+lang:TypeScript+select:content&patternType=standard&sm=1&groupBy=path) for `code host` in existing messaging/content.

Reply:
Exactly my point. In the original code there was a dash, and I removed it. :)

Reply:
I suspected so; that happens to me pretty often, too 😅